patterns/services-three-column-boxed-pricing-table.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Title: Services - Three-column boxed pricing table
+ * Slug: twentytwentyfive/services-three-column-boxed-pricing-table
+ * Categories: call-to-action, banner, featured, services, pricing
```

Comment:
The pricing category does no exist, so please remove it.

Reply:
@carolinan I have removed the pricing category